.prettierignore

```diff
@@ -2,4 +2,6 @@
 !src/*
 resources
 .*
-dist
\ No newline at end of file
+dist
+__snapshots__
+LICENSE
```

Comment:
The Licence file should not be ignored

Reply:
But during format:check we get this error otherwise:
`Error: No parser could be inferred for file: LICENSE`
The job still succeeds but it seems unnecessary to check the format of the license file.

Reply:
If there is no parser for the format check, this should not lead us to ignore the file. 
Let's rather think of ignoring the file in the config of the format checker. Another way would be just to change the LICENCE file to LICENCE.md. I think I forgot to add the .md prefix when I created the file. :-)

Reply:
Isn't the `.prettierignore`-file not already the config file of the formatter for making it ignore certain files?

Reply:
I note that the LICENCE-file was renamed to be a .md-file. We probably could also ignore all files without file extension. But its fine for now.